src/utils/adapters.js

```diff
@@ -80,10 +80,8 @@ export function adaptChainNames(chainNames, blockHeight) {
         chainName.info.active_from,
         blockHeight,
       ),
-      isAuction: isAuction(chainName),
-      price: isAuction(chainName)
-        ? formatAettosToAe(chainName.last_bid.tx.name_fee)
-        : formatAettosToAe(chainName.info.claims[chainName.info.claims.length - 1].tx.name_fee),
+      isAuction: isAuction(chainName.name),
+      price: formatAettosToAe(chainName.info.claims[chainName.info.claims.length - 1].tx.name_fee),
```

Comment:
Apparently there was some change in the mdw since we implemented this code and this code can be used to get the price for both auctioned name and a direct buy

Reply:
Have you tried this? Its a new addition to JS https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at